docs/bn/week06/06-1.md

```diff
@@ -0,0 +1,186 @@
+
+<center>
+<img src="{{site.baseurl}}/images/week06/06-1/O1IN3JD.png" style="zoom: 40%; background-color:#DCDCDC;"/><br>
+<b>চিত্র ১: </b>জিপ কোড সনাক্তকরণে প্রয়োগকৃত একাধিক ক্লাসিফায়ার
+</center>
+
+
+<!--Now to impose the order of the characters. The trick is to utilize a shortest path algorithm. Since we are given ranges of possible characters and the total number of digits to predict, We can approach this problem by computing the minimum cost of producing digits and transitions between digit. The path has to be continuous from the lower left cell to the upper right cell on the graph, and the path is restricted to only contain movements from left to right and bottom to top. Note that if the same number is repeated next to each other, the algorithm should be able to distinguish there are repeated numbers instead of predicting a single digit.-->
+
+
+অক্ষরের ক্রম নিশ্চিত করতে গৃহীত কৌশলটি শর্টেস্ট পাথ এলগোরিদম ব্যবহার করবে। যেহেতু আমাদের দেয়া হয়েছে বিভিন্ন পরিসরে সাজানো সম্ভাব্য অক্ষরসমূহ এবং সর্বমোট নিরীক্ষণীয় সংখ্যার সমষ্টি, আলোচ্য প্রেক্ষাপটে আমাদের সম্ভবপর পন্থা হতে পারে সংখ্যা গঠন করা এবং একের অধিক সংখ্যার মধ্যবর্তী পরিবর্তনের ক্ষেত্রে সর্বনিম্ন পরিব্যয় গণ্না করা । কৌশলের অবলম্বিত পথটি নিম্নবর্তী বাম ঘর থেকে উপরের ডান ঘর পর্যন্ত অবিচ্ছিন্ন ভাবে চলমান হতে হবে । এবং পথটি শুধুমাত্র বাম থেকে ডানের এবং উপরে থেকে নিচের গতিবিধি ধারণ করতে পারবে। এখানে উল্লেখ্য থাকে যে, যদি একই সংখ্যা উপর্যুপরি পুনরাবৃত্তি ঘটে, সেক্ষেত্রে গৃহীত এলগোরিদমটির পুনরাবৃত্ত সংখ্যাকে  একটি একক সংখ্যাকে সনাক্তকরণ থেকে পার্থক্য করতে পারা উচিত।
+
+
+<!--## [Face detection](https://www.youtube.com/watch?v=ycbMGyCPzvE&t=1241s)-->
+
+<!--Convolutional neural networks perform well on detection tasks and face detection is no exception. To perform face detection we collect a dataset of images with faces and without faces, on which we train a convolutional net with a window size such as 30 $\times$ 30 pixels and ask the network to tell whether there is a face or not. Once trained, we apply the model to a new image and if there are faces roughly within a 30 $\times$ 30 pixel window, the convolutional net will light up the output at the corresponding locations. However, two problems exist.-->
+
+## [মুখাবয়ব সনাক্তকরণ](https://www.youtube.com/watch?v=ycbMGyCPzvE&t=1241s)
+কনভলিউশনাল নিউরাল নেটওয়ার্ক সনাক্তকরণের কাজসমূহ ভালো ভাবে সম্পাদন  করতে পারে এবং মুখাবয়ব সনাক্তকরণ এর আওতার বাইরে নয়। কাজটি সম্পন্ন করার জন্য আমরা মুখমণ্ডল সহ এবং মুখমণ্ডল ছাড়া স্থিরচিত্রের একটি ডেটাসেট সংগ্রহ করেছি, যেটার ওপর আমরা কনভলিউশনাল নেট ট্রেইন করব একটি নিরীক্ষণ পরিধি দিয়ে যার আয়তন হবে 30 $\times 30 পিক্সেলস এবং নেটওয়ার্কটিকে প্রশ্ন করা হবে পরীক্ষণিয় স্থিরচিত্রে মুখাবয়ব আছে কি নেই এর উত্তর দিতে। একবার ট্রেইনিং সম্পন্ন করার পর আমরা মডেলটিকে নতুন স্থিরচিত্রের উপর প্রয়োগ করব এবং যদি একটি 30 $\times 30 পিক্সেল পরিধির মধ্যে মুখাবয়বের উপস্থিতি থাকে সেক্ষেত্রে কনভলিউশনাল নেটটি সংশ্লিষ্ট স্থানে আউটপুটটিকে চিহ্নিত করে দিবে। তবে, এখানে দুটো সমস্যা আছেঃ
```

Comment:
Did you use automatic translation?
The LaTeX is broken here as well.
Also, we need someone who *does* speak Bangla to review this PR.

Reply:
No I haven't used automatic translation, I just didn't check with rendered version until today, my bad! 
I have contacted Khalid Saifullah personally to see if he is available to review.